src/styles/page.css

```diff
@@ -3,7 +3,7 @@
   display: flex;
   flex-direction: column;
   align-items: center;
-  padding-bottom: 1.5%;
+  background-color: lightgray!important;
```

Comment:
So lets leave the styling alone for this issue. Lets try and stick to simple PR's that target one specific issue, or at least related ones.

If there are styling issues, lets make a discussion, or an issue surrounding it and come back to it at a later time.